src/audio/google/google_rtc_audio_processing.c

```diff
@@ -424,8 +435,7 @@ static int google_rtc_audio_processing_init(struct processing_module *mod)
 
 	cd->num_aec_reference_channels = CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING_NUM_AEC_REFERENCE_CHANNELS;
 	cd->num_capture_channels = CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING_NUM_CHANNELS;
-	cd->num_frames = CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING_SAMPLE_RATE_HZ /
-		GOOGLE_RTC_AUDIO_PROCESSING_FREQENCY_TO_PERIOD_FRAMES;
+	cd->num_frames = NUM_FRAMES;
```

Comment:
Why not get rid of this and use the define directly?

Reply:
Pre-existing style I didn't bother to change.  Also I guess LINE_LENGTH_AGAIN_AS_THE_KCONFIGS_ARE_A_LOT_OF_TYPING.  But yeah, some cleanup on the comp_data struct is definitely needed.  Once the stream stuff is committed the next batch of cleanup is going to look at state and config handling.